codecov.yml

```diff
@@ -1,3 +1,14 @@
 ignore: 
   - "**/*.pb.go"     # ignore generated protos
   - "**/*.pb.gw.go"  # ignore generated protos
+coverage:
+  range: 60..80
```

Comment:
why an upper bound of 80? or am I misreading this config line?

Reply:
An upper bound of 80 means that code coverage above 80% will be marked as green.
This can be adjusted as the code coverage for the project grows over time.